doc/makefile

```diff
@@ -0,0 +1,196 @@
+# Makefile for Sphinx documentation
```

Comment:
Why're we adding doc stuff which is unused? It's a bit crufty.

Reply:
we have no docs atm however we could/should soon.

Reply:
Instead of adding this cruft now, why not add it then. It just becomes more lines of code to manage and another set of tests to deal with.